manifests/service.pp

```diff
@@ -5,7 +5,7 @@
 
 class datadog_agent::service(
   $service_ensure = 'running',
-  Boolean $service_enable = true,
+  Variant[Boolean, Enum['manual', 'masked', 'delayed']] $service_enable = true,
```

Comment:
The [docs](https://puppet.com/docs/puppet/7/types/service.html#service-attribute-enable) say this should be `mask` instead of `masked`. Which one is the right one?

Reply:
Should be mask, thanks for catching that!